database/seeds/PlaceTableSeeder.php

```diff
@@ -60,13 +62,24 @@ public function run()
             'default' => true,
         ]);
 
+        /* Choose random city to locate places in */
+        $city = factory(City::class)->create([
+            'name' => array_random(self::CITY_NAMES)
+        ]);
+
+        /* Seed users table */
+        $this->call(UserTableSeeder::class);
+
+        /* Seed categories table */
+        $this->call(PlaceCategoryTableSeeder::class);
+
         foreach (self::PLACE_NAMES as $placeName) {
-            $city = factory(City::class)->create([
-                'name' => array_random(self::CITY_NAMES)
-            ]);
             $place = factory(Place::class)->create([
-                'city_id' => $city->id,
+                'city_id'       => $city->id,
+                'creator_id'    => User::orderByRaw('RAND()')->first()->id,
```

Comment:
`User::inRandomOrder()->first();` is more obvious.

Reply:
I agree with you.